internal/pkg/cli/svc_deploy.go

```diff
@@ -135,14 +136,19 @@ func newSvcDeployOpts(vars deployWkldVars) (*deploySvcOpts, error) {
 			}
 			return d, nil
 		},
-		cmd:            exec.NewCmd(),
-		sessProvider:   sessions.NewProvider(),
-		snsTopicGetter: deployStore,
+		newInterpolator: newManifestInterpolator,
```

Comment:
What do you think of just doing
```suggestion
		newInterpolator: manifest.NewInterpolator,
```
and same in `job_deploy.go`

Reply:
they return to different type :)